### ISSUE TITLE
Tests: validate_proxy_slicing: ValueError: cannot set WRITEABLE flag to True of this array

### ISSUE THREAD

**Issue by ignatenkobrain** [nipy/nibabel#370]
```python
======================================================================
ERROR: autogenerated test from validate_proxy_slicing
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/nibabel-2.0.1/nibabel/tests/test_api_validators.py", line 22, in meth
    validator(self, imaker, params)
  File "/builddir/build/BUILD/nibabel-2.0.1/nibabel/tests/test_proxy_api.py", line 159, in validate_proxy_slicing
    assert_array_equal(arr[sliceobj], prox[sliceobj])
  File "/builddir/build/BUILD/nibabel-2.0.1/nibabel/parrec.py", line 581, in __getitem__
    _, slopes, inters = np.broadcast_arrays(fake_data, slopes, inters)
  File "/usr/lib64/python2.7/site-packages/numpy/lib/stride_tricks.py", line 200, in broadcast_arrays
    for array in args]
  File "/usr/lib64/python2.7/site-packages/numpy/lib/stride_tricks.py", line 70, in _broadcast_to
    result.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

I'm not sure, probably bug in NumPy. I'm using NumPy 1.10.1

**Comment by effigies** [comment]
Fixed in #358. If you can use the git repository instead of a release version, that should solve your problem.

**Comment by ignatenkobrain** [comment]
@effigies I'm packaging nibabel for Fedora, so I'd like to use stable version if no critical issues found. I will apply patch from #358 for now. Thanks for very quick response!
